poutyne/framework/model.py

```diff
@@ -1007,7 +1007,7 @@ def _transfer_loss_and_metrics_modules_to_right_device(self):
         if isinstance(self.loss_function, torch.nn.Module):
             self.loss_function.to(self.device)
 
-        for metric in self.batch_metrics:
+        for metric in self.batch_metrics + self.epoch_metrics:
```

Comment:
I prefer this approach as my previous one. Now it's called in the forward pass right ?

Reply:
Yes.

Reply:
Nice.